File: app/src/main/java/com/so/luotk/models/youtubeEx/youtube/playerResponse/Storyboards.java
package com.so.luotk.models.youtubeEx.youtube.playerResponse;

import java.io.Serializable;

public class Storyboards implements Serializable {
    private PlayerStoryboardSpecRenderer playerStoryboardSpecRenderer;

    public PlayerStoryboardSpecRenderer getPlayerStoryboardSpecRenderer() {
        return playerStoryboardSpecRenderer;
    }

    public void setPlayerStoryboardSpecRenderer(PlayerStoryboardSpecRenderer playerStoryboardSpecRenderer) {
        this.playerStoryboardSpecRenderer = playerStoryboardSpecRenderer;
    }

    @Override
    public String toString() {
        return
                "Storyboards{" +
                        "playerStoryboardSpecRenderer = '" + playerStoryboardSpecRenderer + '\'' +
                        "}";
    }
}

File: app/src/main/java/com/so/luotk/models/youtubeEx/newModels/Storyboards.kt
package com.so.luotk.models.youtubeEx.newModels

import com.google.gson.annotations.SerializedName

class Storyboards {
    @SerializedName("playerStoryboardSpecRenderer")
    var playerStoryboardSpecRenderer: PlayerStoryboardSpecRenderer? = null
    override fun toString(): String {
        return "Storyboards{" +
                "playerStoryboardSpecRenderer = '" + playerStoryboardSpecRenderer + '\'' +
                "}"
    }
}